### PR TITLE
Disable plugin crawling from GitHub and RubyGems

### PR DESCRIPTION
**Pull request by dmikurube**
Entirely disabling to crawl plugins. This update will be deployed to Heroku automatically. @sakama @kamatama41

### CODE REVIEW

**Review by sakama — APPROVED** [review]
LGTM

File: server.rb [hunk 1]
```diff
@@ -1,25 +1,9 @@
 require 'sinatra'
 require 'sinatra/streaming'
-require_relative 'render'
 
 set :server, 'puma'
 set :port, (ENV['PORT'] || 8580).to_i
 
-get '/update' do
```
Comment (kamatama41) [inline server.rb:8]: Although I'm not sure how this endpoint is called regularly, should it be left as empty method until stop calling the cron like job? (If it is already stopped, never mind.)

Comment (dmikurube) [reply server.rb:8]: The cron job has been stopped, and actually the past cron job was not via HTTP. (Calling `render.rb` directly from Heroku's feature.)

**Review by kamatama41 — APPROVED** [review]
Got it, LGTM 👍

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**